### PR TITLE
Fixed publishing of VS15 symbols for CI builds

### LINKED ISSUE #3185: Fix publishing of VS15 symbols for CI builds

### PR DESCRIPTION
**Pull request by alpaix**
Fixes NuGet/Home#3185
- Added a parameter pointing  to the right version of NuGet.Core for pdb publishing
- Added pdb to the command line package nuspec
- Minor script fixes

//cc @drewgil @rrelyea @joelverhagen @emgarten

### CODE REVIEW

File: test/TestExtensions/GenerateTestPackages/project.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ﻿{
   "dependencies": {
-    "NuGet.Core": "2.8.5"
+    "NuGet.Core": "2.13.0-rtm-824"
```
Comment (joelverhagen) [inline test/TestExtensions/GenerateTestPackages/project.json:3]: Has this been run through the E2E tests yet?

File: src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec [hunk 2]
```diff
@@ -13,5 +13,6 @@
   </metadata>
   <files>
       <file src="NuGet.exe" target="tools" />
+      <file src="NuGet.pdb" target="tools" />
```
Comment (joelverhagen) [inline src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec:16]: PDB is normally in a .symbols package, right?

Comment (alpaix) [inline src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec:16]: Tested with: 

``` sh
nuget pack .\src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.nuspec -outputdirectory .\Nupkgs\ -symbols
```

Here is the output:

```
Attempting to build package from 'NuGet.CommandLine.nuspec'.
Successfully created package '.\Nupkgs\NuGet.CommandLine.3.5.0-rc1.nupkg'.
Successfully created package '.\Nupkgs\NuGet.CommandLine.3.5.0-rc1.symbols.nupkg'.
```

The .sumbols package does have .pdb in tools after that.

Comment (joelverhagen) [inline src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec:16]: And the one without `.symbols.`? Should it have a PDB or not?

File: test/TestExtensions/GenerateTestPackages/project.json (hunk 1 above)
Comment (alpaix) [inline test/TestExtensions/GenerateTestPackages/project.json:3]: Ran locally. Running now on CI on my dev-\* branch.

File: src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec (hunk 2 above)
Comment (alpaix) [inline src/NuGet.Clients/NuGet.CommandLine/NuGet.CommandLine.nuspec:16]: It didn't before and won't after the change. nuget.exe is smart enough not to include symbols into regular package :smiley_cat:

**Comment by joelverhagen** [comment]
If E2E passes, :shipit:

**Pull request merged**